### ISSUE TITLE
Renaming columns in DataFramesMeta with spaces

### ISSUE THREAD

**Issue by jacobcvt12** [JuliaData/DataFramesMeta.jl#93]
Cross-posting from https://discourse.julialang.org/t/renaming-columns-in-dataframesmeta-with-spaces/11101

I’ve created the following dataset:

```
d = DataFrame(x=randn(100))
d[Symbol("Y 2")] = randn(100)
```

I can rename the second column via

```
rename!(d, Symbol("Y 2") => :Y)
```

But I would like to rename it via a DataFramesMeta chain.

I can do

```
@linq d |> select(X=:x)
```

But

```
@linq d |> select(Y=Symbol("Y 2"))
```

drops the column Y 2 and adds the column Y with only one row with the value “Y 2”

Additionally,

```
@linq d |> select(X=Symbol("x"))
```

Does something similar. Anyway to get around this, or do I need to use rename!?

**Comment by pdeffebach** [comment]
I think the issue is [here](https://github.com/JuliaStats/DataFramesMeta.jl/blob/b016059cce76773ca0382dbaed15eb13e1d71f2d/src/DataFramesMeta.jl#L640), which goes 

```
function expandargs(e::Expr)
    if e.head == :quote
        Expr(:kw, e.args[1], e)
    else
        replace_equals_with_kw(e)
    end
end
```

If we look at the structure of `:Y2:` vs `Symbol("Y 2")` we see that the second one's head is not `:quote`

```
e1 = :(:Y2)
e1.head 
> :quote

e2 = :(Symbol("Y 2"))
e2.head
> :call
```

**Comment by sswatson** [comment]
One solution here is to protect `Symbol("Y 2")` with `cols`:

```julia
@linq d |> select(Y=cols(Symbol("Y 2")))
```

**Comment by pdeffebach** [comment]
Closing in favor of #204 . 

Though some of these problems were probably fixed with #171
